service/request/index.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+import got from 'got';
+import merge from 'lodash.merge';
+
+function createRequestService() {
+    function post(options) {
+        let opts = {
```

Comment:
Duplication (DRY)

post, get, and patch repeat near-identical blocks differing only by method.
This increases maintenance friction if defaults change (e.g. add timeout or user-agent).
Consider extracting a shared builder or using got.extend?